### ISSUE TITLE
Request: Home assistant Integration

### ISSUE THREAD

**Issue by husnainali09** [derkalle4/python3-idotmatrix-client#34]
Thank you for your project. A HASS integration will make this so much more viable to use please.

**Comment by derkalle4** [comment]
Hi @husnainali09 

many thanks for your first issue :) due to long-covid I have no energy to work on this. Unfortunately I cannot see when I have the time to do so (but I want to do this integration, for sure).

Anyhow, this would be a standalone plugin. This client is only for linux/windows. I do have a library which should make the include pretty easy: https://github.com/derkalle4/python3-idotmatrix-library 

I'll keep you updated and this ticket active until I have started implementation.

Kind Regards,
Kalle

**Comment by Dokport** [comment]
Nice project. +1. HASS integration would be amazing!

**Comment by derkalle4** [comment]
Yeah, whenever I am back on track ;) or someone adds a push-request.

**Comment by clach04** [comment]
https://github.com/derkalle4/python3-idotmatrix-client/issues/30

**Comment by ebuy05** [comment]
+1 for the HA integration on HACS. I've been trying to run on linux (HA Terminal) and I keep getting errors when trying to build the GUI (I couldn't install PyQt5 or PyQt6)...
